slsDetectorSoftware/src/slsDetector.cpp

```diff
@@ -948,124 +940,80 @@ int slsDetector::updateDetector() {
-        names.emplace_back("rx_udpport");
-        names.emplace_back("rx_udpport2");
-        names.emplace_back("rx_udpip");
-        names.emplace_back("rx_udpmac");
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("vhighvoltage");
-        names.emplace_back("trimen");
-        names.emplace_back("iodelay");
-        names.emplace_back("tengiga");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectorip");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpport2");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpmac");
+        base.emplace_back("trimen");
+        base.emplace_back("iodelay");
+        base.emplace_back("tengiga");
         break;
     case JUNGFRAU:
-        names.emplace_back("detectormac");
-        names.emplace_back("detectormac2");
-        names.emplace_back("detectorip");
-        names.emplace_back("detectorip2");
-        names.emplace_back("rx_udpport");
-        names.emplace_back("rx_udpport2");
-        names.emplace_back("rx_udpip");
-        names.emplace_back("rx_udpip2");
-        names.emplace_back("rx_udpmac");
-        names.emplace_back("rx_udpmac2");
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("powerchip");
-        names.emplace_back("vhighvoltage");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectormac2");
+        base.emplace_back("detectorip");
+        base.emplace_back("detectorip2");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpport2");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpip2");
+        base.emplace_back("rx_udpmac");
+        base.emplace_back("rx_udpmac2");
+        base.emplace_back("powerchip");
         break;
     case CHIPTESTBOARD:
-        names.emplace_back("detectormac");
-        names.emplace_back("detectorip");
-        names.emplace_back("rx_udpport");
-        names.emplace_back("rx_udpip");
-        names.emplace_back("rx_udpmac");
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("vhighvoltage");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectorip");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpmac");
         break;
     case MOENCH:
-        names.emplace_back("detectormac");
-        names.emplace_back("detectorip");
-        names.emplace_back("rx_udpport");
-        names.emplace_back("rx_udpip");
-        names.emplace_back("rx_udpmac");
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("powerchip");
-        names.emplace_back("vhighvoltage");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectorip");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpmac");
+        base.emplace_back("powerchip");
```

Comment:
This is a bug I introduced sometime back. Could you please remove the powerchip command for Moench?

Reply:
done